### DISCUSSION TITLE
目前表单生命周期是一个黑盒，不太清楚生命周期内是如何运作的

### DISCUSSION THREAD

**Discussion by LouisaNikita** [alibaba/formily#1580] [Q&A 2.x] [answered]
What problem does this feature solve?
目前表单生命周期是一个黑盒，不太清楚生命周期内是如何运作的

What does the proposed API look like?
期待有一个生命周期介绍，每个节点都做了什么

之前回复核心文档在子站里找，子站里提到只提到lifeCycle,但是lifeCycle具体有哪些做什么，没有详细介绍，我还是翻代码翻到了lifeCycle具体有哪些，但是每个具体做什么还没看

**Comment by janryWang** [comment, marked as answer]
https://core.formilyjs.org/api/entry/form-effect-hooks
https://core.formilyjs.org/api/entry/field-effect-hooks
